src/deploy/functions/params.ts

```diff
@@ -444,6 +509,50 @@ async function promptParam(
   assertExhaustive(param);
 }
 
+async function promptList(
+  param: ListParam,
+  projectId: string,
+  resolvedDefault?: string[]
+): Promise<string[]> {
+  if (!param.input) {
+    const defaultToText: TextInput<string> = { text: {} };
+    param.input = defaultToText;
+  }
+  let prompt: string;
+
+  if (isSelectInput(param.input)) {
+    throw new FirebaseError("List params cannot have non-list selector inputs");
+  } else if (isListSelectInput(param.input)) {
+    prompt = `Select a value for ${param.label || param.name}:`;
+    if (param.description) {
+      prompt += ` \n(${param.description})`;
+    }
+    prompt += "\nSelect an option with the arrow keys, and use Enter to confirm your choice. ";
+    return promptSelectMultiple<string>(
+      prompt,
+      param.input,
+      resolvedDefault,
+      (res: string[]) => res
+    );
+  } else if (isTextInput(param.input)) {
+    prompt = `Enter a list of strings for ${param.label || param.name}:`;
```

Comment:
can we expand the prompt to help instruct users how they can input the delimeter-separated list input?